src/parsers/manifestjson.js

```diff
@@ -180,6 +191,15 @@ export default class ManifestJSONParser extends JSONParser {
     });
   }
 
+  validateExists(filePath, type) {
```

Comment:
I'd call this `validateFileExistsInPackage` or just `validateFileExists` since that's what it does.

Reply:
okey doke